ThirdPartyAdapters/vungle/vungle/src/main/java/com/google/ads/mediation/vungle/VungleMediationAdapter.java

```diff
@@ -337,7 +335,8 @@ public void run() {
 
   @Override
   public void onAdViewed(String placementId) {
-    // "no-op , to be mapped to respective adapter events in future release"
+    mMediationRewardedAdCallback.reportAdImpression();
+    mMediationRewardedAdCallback.onVideoStart();
```

Comment:
Not sure if we should keep the sequence to call onVideoStart() first, and then reportAdImpression().

Reply:
@HuFangshuai I can revert it back to the previous onVideoStart and then reportAdImpression, besides keeping continuity is there any logical reason that you know of, of why we need to keep this order?

Reply:
Not sure, just found on the left side as the original implementation, we called them in sequence of onAdOpened()->onVideoStart()-> reportAdImpression().